packages/cli/test/functional/test_site_convert/test_basic_convert/expected/index.html

```diff
@@ -6,7 +6,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="generator" content="MarkBind 4.0.2">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Landing Page</title>
+  <title></title>
```

Comment:
We should look into removing this empty title tag as it holds no value, either in a separate PR or in this one.

Reply:
Good point! I was planning to open a new issue / PR for cleaning up the default / minimal site templates, so I will include this as part of the new issue :+1: